docs/source/options/general_encryption_options.rst

```diff
@@ -12,7 +12,12 @@ General encryption options
 
 --clear_lead <seconds>
 
-    Clear lead in seconds if encryption is enabled.
+    Shaka Packager does not support partial encrypted segments, all the 
+    segments including the partial segment overlapping with the initial 
+    'clear_lead' seconds are not encrypted, with all the following segments 
+    encrypted. For example, if segment_duration is greater than 'clear_lead', 
+    the first segment is not encrypted, then only the first segment is
```

Comment:
Sorry for typo in my previous comments (was distracted).

I meant to keep the sentence "Clear lead in seconds if encryption is enabled." It should be something like:

    Clear lead in seconds if encryption is enabled.
    Shaka Packager does not support partial encrypted segments, all the 
    segments including the partial segment overlapping with the initial 
    'clear_lead' seconds are not encrypted, with all the following segments 
    encrypted. If segment_duration is greater than 'clear_lead', then only the
    first segment is not encrypted.
    Default: 5

Reply:
No problem, I pushed your text now.